src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -95,6 +95,8 @@ private enum TimeZoneInfoResult {
         private static readonly TransitionTime c_transition7_15  = TransitionTime.CreateFixedDateRule(new DateTime(1, 1, 1), 07, 15);
         private static readonly TransitionTime c_transition10_15 = TransitionTime.CreateFixedDateRule(new DateTime(1, 1, 1), 10, 15);
         private static readonly TransitionTime c_transition12_15 = TransitionTime.CreateFixedDateRule(new DateTime(1, 1, 1), 12, 15);
+
+        private const string c_timeZoneDirectory = "/usr/share/zoneinfo";
```

Comment:
> Just verifying that this is the right path on all relevant flavors of Unix?

Reading the 'man tzset' doc - 
>       The  system  timezone  directory  used  depends on the (g)libc version.
       Libc4 and libc5 use /usr/lib/zoneinfo, and, since libc-5.4.6, when this
       doesn't  work, will try /usr/share/zoneinfo.  Glibc2 will use the envi‐
       ronment variable TZDIR, when that exists.  Its default depends  on  how
       it was installed, but normally is /usr/share/zoneinfo.

I'm assuming we only care about Glibc2.  Is that correct?  If so, it looks like I should add a check for *TZDIR*.